topics/single-cell/tutorials/EBI-retrieval/tutorial.md

```diff
@@ -329,17 +348,17 @@ If you chose **Loom** format and you need to convert your file to other datatype
 
 > <hands-on-title>Create AnnData object</hands-on-title>
 >
-> {% tool [Scanpy Read10x](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_read_10x/scanpy_read_10x/1.8.1+galaxy0) %} with the following parameters:
+> {% tool [Scanpy Read10x](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_read_10x/scanpy_read_10x/1.8.1+galaxy9) %} with the following parameters:
+>  Make sure you are using version **1.8.1+galaxy9** of the tool (change by clicking on {% icon tool-versions %} Versions button)
 >    - *"Expression matrix in sparse matrix format (.mtx)"*: `Human Cell Atlas Matrix Downloader on matrix.mtx`
 >    - *"Gene table"*:  `Human Cell Atlas Matrix Downloader on genes.tsv`
 >    - *"Barcode/cell table"*: `Human Cell Atlas Matrix Downloader on barcodes.tsv`
 >    - *"Cell metadata table"*: `Human Cell Atlas Matrix Downloader on exp_design.tsv`
 {: .hands_on}
 
-
 > <tip-title>Flagging genes by using AnnData Operations</tip-title>
 >
-> After you create AnnData file, you can additionally use the {% tool [AnnData Operations](toolshed.g2.bx.psu.edu/repos/ebi-gxa/anndata_ops/anndata_ops/1.8.1+galaxy92) %} tool before downstream analysis. It's quite a useful tool since not only does it flag mitochondrial genes, but also automatically calculates a bunch of metrics, such as `log1p_mean_counts`, `log1p_total_counts`, `mean_counts`, `n_cells`, `n_cells_by_counts`, `n_counts`, `pct_dropout_by_counts`, and `total_counts`.
+> After you create AnnData file, you can additionally use the {% tool [AnnData Operations](toolshed.g2.bx.psu.edu/repos/ebi-gxa/anndata_ops/anndata_ops/1.8.1+galaxy92) %} tool (note the version **1.8.1+galaxy9**) before downstream analysis. It's quite a useful tool since not only does it flag mitochondrial genes, but also automatically calculates a bunch of metrics, such as `log1p_mean_counts`, `log1p_total_counts`, `mean_counts`, `n_cells`, `n_cells_by_counts`, `n_counts`, `pct_dropout_by_counts`, and `total_counts`.
```

Comment:
is the __2__ missing here or intentional for the `...+galaxy9`

Reply:
uhh, great catch, thanks! It was supposed to help the users and it would confused them even more!